.gitignore

```diff
@@ -1,2 +1,6 @@
 #ignore all files in .idea
```

Comment:
Please help ignore it.
![image](https://github.com/user-attachments/assets/176d29e2-d79d-4abc-b614-cae11e267ee5)